sphinx_needs/libs/html/jquery-3.6.0.min.js

```diff
@@ -0,0 +1,2 @@
+/*! jQuery v3.6.0 | (c) OpenJS Foundation and other contributors | jquery.org/license */
```

Comment:
Is this needed in SN?

There is a Sphinx-Jquery extension, which could care about it.

Reply:
Just came across the extension so I will remove it now.